modules/identify-vulnerabilities-in-nodes.adoc

```diff
@@ -17,6 +17,7 @@ The identified vulnerabilities include vulnerabilities in:
 * {product-title} can identify vulnerabilities in the following Linux kernels:
 ** Amazon Linux 2
 ** CentOS
+** CoreOS
```

Comment:
If this is RHCOS, can you match `:op-system-first:` from line 12 in [common-attributes.adoc](https://github.com/openshift/openshift-docs/blob/5764b31f2cec223a3e04b87cd63258dfe4565940/modules/common-attributes.adoc#L12)?  If it's the EOLed Container Linux (had to google it), maybe "Container Linux, formerly CoreOS".

It's outside the scope of your update, but aren't these Linux distributions rather than Linux kernels?  Is it "...can identify kernel vulnerabilities in the following Linux distributions:" on line 17?